src/es/components/organisms/footer/default-/default-.html

```diff
@@ -1,40 +1,125 @@
-<o-footer namespace="footer-default-">
+<o-footer namespace="footer-default-" no-details-mobile>
+    <style>
+        :host .invert {
+            padding-top: 160px;
```

Comment:
It is good practice to introduce a new variable with a fix value as default. Later, if some changes are coming, you can do it in variableCustom.css then to override.
```suggestion
            padding-top: var(--footer-invert-padding-top, 160px);
```